<launch>


    <!--Neptus addr: Address of computer running netpus -->
    <!--bridge addr: Address of computer running this node -->
    <!--bridge port: Port neptus sends data to -->
  

  <arg name="neptus_addr" default="127.0.0.1"/>
  <arg name="bridge_addr" default="127.0.0.1"/>
  <arg name="bridge_port" default="6002"/>
  <arg name="node_name" default="imc_bridge"/>
  <arg name="imc_system_name" default="imc_ros_bridge"/>

  <!-- 4=imc_ros_bridge, 5=sam, 6=lolo -->
  <arg name="imc_id" default="32886"/>
  <arg name="imc_src" default="4"/>

  <node pkg="imc_ros_bridge" exec="bridge_node" name="$(var node_name)" output="screen" launch-prefix='gdbserver localhost:3000' namespace="imc">
    <param name="neptus_addr" value="$(var neptus_addr)"/>
    <param name="bridge_addr" value="$(var bridge_addr)"/>
    <param name="bridge_port" value="$(var bridge_port)" type="str"/>
    <param name="system_name" value="$(var imc_system_name)"/>
    <param name="imc_id" value="$(var imc_id)"/>
    <param name="imc_src" value="$(var imc_src)"/>
  </node>

</launch>
